CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Change Log
+
+## 0.11.0
+
+- Feature: Automatically download sauce connect client with version `4.5.4`
+- Enhancement: Upgrade joi to 14.3.1
+- Enhancement: Update sauce object schema to add `launchSauceConnect` flag
+- Fix: when `launchSauceConnect = true` and tunnelIdentifier is also set
```

Comment:
@xwa130saucer these are not fixes, as they are not bugs. Maybe just update the line above with "added validation rules for new launchSauceConnect option"?

Reply:
Does this count as Enhancement? I'm just trying to classify them into each bucket...

Reply:
All of this is associated with a new Feature, which is SC integration

Reply:
Got it